src/main/java/org/opentest4j/MultipleFailuresError.java

```diff
@@ -90,11 +92,13 @@ private static String pluralize(int count, String singular, String plural) {
 		return count == 1 ? singular : plural;
 	}
 
-	private static String nullSafeMessage(AssertionError failure) {
+	private static String nullSafeMessage(Throwable failure) {
+		if (failure == null) {
```

Comment:
Again this change was done to accommodate representing the  multiple failures that the JUnit5 vintage engine must record.

In that context a failure which is represented by the platform's TestExecutionResult can legally(according to the doc) contain null Throwables, where without the above check the above code would crash with an NPE if such results were involved in a multiple failure. So thinking in a fail-safe mode I added the check.

_Now that I think about it a bit more, we could move this check where I add the values to the MultipleFailuresError and add eg a Throwable with the empty null message. This make the distance the null value travels shorter. But it seems too much detailed thinking for this._

_What do you think @kcooney?_